aria_data_loaders/aria_data_utils/detector_wrappers/hod_wrapper.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import model as hand_object_detector
+import numpy as np
+import torch
+from aria_data_utils.detector_wrappers.generic_detector_interface import GenericDetector
+from aria_data_utils.perception.object_in_hand_detector import setup_hod_network
+from hand_object_detector.faster_rcnn.faster_rcnn import _fasterRCNN
+
+
+class ObjectInHandDetectorWrapper(GenericDetector):
```

Comment:
This needs basic testing before we merge it in.

Reply:
Maybe remove this and only test+merge `object_in_hand_detector` in this PR.

Reply:
Let's remove hand object detector from this PR & have a separate PR for this change?

Reply:
moved to https://github.com/facebookresearch/spot-sim2real/pull/114